src/kOS/Module/kOSProcessor.cs

```diff
@@ -240,11 +240,14 @@ public ModifierChangeWhen GetModuleCostChangeWhen()
             return ModifierChangeWhen.FIXED;
         }
 
+        public float additionalMass = 0F;
         private void UpdateCostAndMass()
         {
+            
             float spaceDelta = diskSpace - baseDiskSpace;
             additionalCost = (float)System.Math.Round(spaceDelta * diskSpaceCostFactor, 0);
             additionalMass = spaceDelta * diskSpaceMassFactor;
+            additionalMassGui = spaceDelta * diskSpaceMassFactor * 1000;
```

Comment:
I'd change this to be
```
additionalMassGui = additionalMass * 1000;
```
just to be more concise, and prevent a later change from only modifying one of the assignments.